frontend/src/containers/EditCollective.js

```diff
@@ -0,0 +1,370 @@
+              )}
+              {highlightField === 'mission' && (
+                <CustomTextArea cols='29' {...this.props} value={ highlightValue } onChange={ this.onChangeHighlightValueRef } maxLength={100}/>
+              )}
+              {highlightField === 'name' && (
+                <Input {...this.props} value={ highlightValue } handleChange={ this.onChangeHighlightValueRef } maxLength={255}/>
+              )}
+              {highlightField === 'website' && (
+                <Input {...this.props} value={ highlightValue } handleChange={ this.onChangeHighlightValueRef } maxLength={255}/>
+              )}
+              {highlightField === 'whyJoin' && (
+                <CustomTextArea cols='29' {...this.props} value={ highlightValue } onChange={ this.onChangeHighlightValueRef } />
+              )}
+              {highlightField === 'image' && (
+                <ImagePicker
+                  uploadOptionFirst
+                  label='Choose media source'
+                  className="logo"
+                  dontLookupSocialMediaAvatars
+                  presets={ PRESET_LOGOS }
+                  src={ highlightValue }
+                  handleChange={ this.onChangeHighlightValueRef } 
+                  {...this.props} />
+              )}
+            </Modal>
+          </Overlay>
+        )}
+        {highlights.map(highlight => {
+          return <Highlight { ...highlight } onClick={ () => this.onClickHighlight(highlight) } />
+        })}
+      </div>
+    )
+  }
+
+  componentDidMount() {
+    this.updateHighlights();
+    window.addEventListener('resize', this.lazyUpdateHighlights);
+    this.forceUpdate();
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.lazyUpdateHighlights);
+  }
+
+  onPublish() {
+    const { originalGroup, updateGroup } = this.props;
+    const { fields } = this.state;
+    const updatedFields = {};
+    Object.keys(fields).forEach(fieldName => {
+      if (originalGroup[fieldName] !== fields[fieldName]) {
+        updatedFields[fieldName] = fields[fieldName];
+      }
+    })
+    if (Object.keys(updatedFields).length) {
+      updateGroup(originalGroup.id, updatedFields)
+      .catch(error => notify('error', error.message));
+    }
+  }
+
+  onCloseModal(e) {
+    const targetClassName = e.target.className;
+    if (targetClassName === 'EditCollective-Overlay' || targetClassName === 'OnBoardingButton' || targetClassName === '-close' ) {
+      this.setState({showModal: false});
+    }
+  }
+
+  onChangeHighlightValue(newValue) {
+    const { highlightField, fields } = this.state;    
+    fields[highlightField] = newValue;
+    this.setState({fields: fields});
+    this.updateHighlights();
+  }
+
+  onClickHighlight(highlight) {
+    this.setState({
+      showModal: true,
+      highlightLabel: highlight.label,
+      highlightField: typeof highlight.field === 'funciton' ? highlight.field(this.state) : highlight.field,
```

Comment:
typo `funciton` will fix in https://github.com/OpenCollective/opencollective-website/pull/424 (where the error is spotted by new eslint rules)